### PR TITLE
Feature: Favorites is finished, just gotta put it on the server and test it

### PR DESCRIPTION
**Pull request by arindavis**

### CODE REVIEW

**Review by samjohnson097 — APPROVED** [review]

File: client/Components/Home/Favorites/FavoritesScreen.js [hunk 1]
```diff
@@ -1,22 +1,86 @@
...
+    axios.get(`http://ec2-3-134-108-148.us-east-2.compute.amazonaws.com:3001/?user_id=${user.id}`)
+      .then(({data}) => {
+        setRealData(data);
+      });
+  };
+
+  const list = () => {
+
+    return realData.map((fav) => {
+      console.log('FAVE', fav);
```
Comment (jstrzesz) [inline client/Components/Home/Favorites/FavoritesScreen.js:30]: remove

Comment (arindavis) [reply client/Components/Home/Favorites/FavoritesScreen.js:30]: you mean the console.log right?

File: client/Components/Root/Login.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ const LoginModal = ({ navigation }) => {
   };
 
   const onSignUp = () => {
-    axios.post('http://ec2-52-15-187-36.us-east-2.compute.amazonaws.com:3001/users/create',
+    axios.post('http://ec2-3-134-108-148.us-east-2.compute.amazonaws.com:3001/users/create',
       {username, password})
       .then(onSignIn)
       .catch(err => console.log(err));
```
Comment (jstrzesz) [inline client/Components/Root/Login.js:39]: change to console.warn

File: client/Components/Settings/SettingsScreen.js [hunk 3]
```diff
@@ -28,6 +29,23 @@ const SettingsScreen = ({navigation, route}) => {
     }
   };
 
+  // const sound1 = new Sound(require('./assets/SolXRloop.wav'),
```
Comment (jstrzesz) [inline client/Components/Settings/SettingsScreen.js:32]: is this old code or future code? If it's code that will be used later, label it with // TODO:

File: server/app.js [hunk 4]
```diff
@@ -28,6 +28,54 @@ app.use('/nasa', nasaRouter);
 //   next(createError(404));
 // });
 
+
```
Comment (jstrzesz) [inline server/app.js:31]: label code with // TODO: if code will be used later or remove if it is no longer needed

File: client/Components/Settings/SettingsScreen.js (hunk 3 above)
Comment (arindavis) [reply client/Components/Settings/SettingsScreen.js:32]: future code. it's not fully implemented yet

File: server/routes/users.js [hunk 5]
```diff
@@ -50,26 +50,42 @@ router.post('/login', (req, res) =>{
...
+      }).then(() => res.send(`Deleted ${title} from your favorites.`));
     } else {
-      Iotd.create({url, date, user_id: id}).then(() => res.sendStatus(201));
+      Iotd.create({url, title, explanation, user_id }).then(() => res.status(201).send(`Added ${title} to your favorites.`));
     }
   });
 });
 
+router.get('/iotd', (req, res)=>{
+  console.log(`here's the data from params: ${req.query.user_id}`)
```
Comment (jstrzesz) [inline server/routes/users.js:74]: remove

**Review by jstrzesz — CHANGES_REQUESTED** [review]
see comments

File: server/routes/users.js (hunk 5 above)
Comment (arindavis) [reply server/routes/users.js:74]: changes are in

**Review by davidsosa22 — APPROVED** [review]

**Pull request merged**